### DISCUSSION TITLE
POLL: Who should enable or create KTON-RING liquidity on the Darwinia chain?

### DISCUSSION THREAD

**Discussion by Laki389** [ktondao/KtonDAO#34] [Polls]
I ask you before answering the question to think about how it is correct from a philosophical/moral/ethical point of view.
If that is possible.

**Problem**

Snowswap's current KTON-RING LP farmer incentive will end in 43 days.
If no solution is found in the meantime, liquidity will disappear, which IMHO is bad for several reasons:

- there is no other DEX or CEX that allows native RING and KTON trading

- Darwinia chain, IMHO should be used as much as possible and increase the number of users, active participants, active addresses, etc.

**Solution** 

We will come to the solution if we first ask ourselves: Who should enable or create KTON-RING liquidity on the Darwinia chain?

Please think carefully and then cast your vote for one of the options.

**Comment by Laki389** [comment]
Now that I've voted, I'd like to argue my chosen option that both DAOs should provide liquidity equally.

The way it could be done.

https://darwinia.subscan.io/account/0x6d6f646c64612f74727372790000000000000000
RING DAO has 779 KTON.
I think it will be easier to secure support from RING DAO voters, instead of that KTON being donated to KTON DAO, that KTON still being owned by RING DAO.
RING DAO would provide initial liquidity of 779 KTON and 779 000 RING.
On Snowswap or maybe some other option like HELIX DEX or maybe something third.

Now there would already be solid liquidity, but IMHO insufficient.
This is where KTON DAO enters the scene.
Of the roughly 84,000 RING per day that goes to KTON DAO and 100% goes to KTON stakers, 20% or say 16k per day would go to liquidity.
How this would be done I leave it up to the Itering developers.
From the already solid liquidity provided by RING DAO, with 16k RING per day it could theoretically add 8 KTON and 8000 RING each.
In theory, a new 800 KTON and 800 000 RING would be provided in just 100 days.

Here I would like to emphasize and point out to KTON hodlers that due to the v2 migration, a solid percentage of KTON will remain unmigrated, so that for the same period of 100 days it is even possible and quite certain that they will still have higher staking rewards than in v1 despite that 16k which 100 days would go towards our goal.

A new survey could be done in the future, how much liquidity as DAOs do we think is enough?
Do we want 2000 KTON and 2 000 000 RING?
Maybe more?
Maybe less?
But about that later. :+1:

**Comment by SasoLithops** [comment]
Just reading through this. Will comment later just one important mistake:

_20% or say 16k per day
 would go to liquidity_

Not liquidity but treasury.

Only certain percentage of those funds would go to farming incentives. KTON DAO will not be able to actually provide liquidity at start imho, as how it looks now building a KTON DAO treasury is a first priority.

**Reply by Laki389** [reply]
> Just reading through this. Will comment later just one important mistake:
> 
> _20% or say 16k per day would go to liquidity_
> 
> Not liquidity but treasury.
> 
> Only certain percentage of those funds would go to farming incentives. KTON DAO will not be able to actually provide liquidity at start imho, as how it looks now building a KTON DAO treasury is a first priority.


If I had hired an English native speaker, a proofreader, a lawyer and a developer and if the text had not been written at 5 a.m., I agree that it would have been better written, but I think anyone who wants it will understand me. :smiley: :kissing_heart:

Ok, let's specify in KTON DAO Treasury.
80k+ RING everyday.
Currently, everything is going for staking.
If 80% would go for staking and 20% for liquidity, with 16k per day for 100 days, in theory, 800 KTON will be bought and together with the remaining 800,000 RING could be put into liquidity once.

Of course, if KTON DAO members decide, they can contribute to liquidity with incentives for KTON - RING LP farmers.
That is also an option.
I think 4000 RING per day would be enough, ie less than 5%.
But it would be all the time, every day.
With the first option, in 100 days in theory 800 KTON and 800 000 RING would be bought, it would be one time.
The second option is always, forever.
But that is of course up to KTON DAO to decide. :+1:

**Comment by nblogist** [comment]
IMO it won't be bad for Ring DAO to support more in the start but eventually when KTON DAO is more stable - it should be more equal.

**Comment by 2xJDubs** [comment]
I have voted that both DAOs should provide to liquidity equally but not in the method that some may think.  I think it would be good to use the KTON in the RINGDAO treasury to support the liquidity on Snowsawp.xyz, it is our only option for a DEX right now, but I also think that farming rewards should be provided keep the KTON holders engaged in providing liquidity.  I don't think it is entirely on the DAOs to provide liquidity as said liquidity benefits holders giving them a place to buy or sell KTON.  Now I think if RINGDAO provides liquidity then KTONDAO can take some of the staking rewards to provide farm rewards.  It will take some time to build up a good amount to support the farm as we saw it took 1.4M RING initially to establish the pool of 1700 KTON so we will need around 800K or 900K RING to attract a good 1000 KTON to support what RINGDAO supplies.

My 2cents.

**Comment by sonoferin2000** [comment]
what about dawinia treasury itself being promoted as a possible source of funding and  chipping  in?,  already they get the lions share of emissions,im not seeing that option being put forward at all. I'm with Saso on this, DAO's need to work on building up a treasury for themselves.

**Comment by hackfisher** [comment]
We can initial a proposal to use KtonDAO treasury for KTON-RING liquidity on Darwinia chain.

I would also suggest talking to Helixbox team, as they deployer Uniswap V3 version AMM and are going to work on Balancer AMM too, this will provide better choice for pool selections.

Also the support for the ui https://swap.helix.box/ is still limited due to the temp unavailable for a Smart Order Router, but support/integation for KTON-RING liquidity should not be a problem for them. cc @hujw77 

https://www.tally.xyz/gov/ktondao/treasury

https://explorer.darwinia.network/address/0x08837De0Ae21C270383D9F2de4DB03c7b1314632

**Reply by sonoferin2000** [reply]
both daos should contribute equally but  rather then constantly having to encentivise LP to simply rent liquidity, one idea worth considering is starting a POL, a protocol owned liquidity, it would'n't be something we could do all at once, but it is a goal that we can incrementally work towards. there are a lot of benefits to POL, it is worth considering in my opinion.
